src/components/Header/MenuButton.component.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Icon = styled.svg`
+    flex: none;
+    width: 25px;
+    height: 25px;
+`;
+
+const Button = styled.button`
+    background-color: transparent;
+    color: transparent;
+    /* border: 2px solid yellow; */
+    /* position: absolute; */
+    /* left: 25px;
+    top:18px; */
+    border: none;
+    display: inline-flex;
+    position: relative;
+    align-items: center;
+    margin: 0px 25px 0px 15px;
+
+    @media(max-width: 640px) {
+        margin: 0px 0px 0px 10px;
+    }
+`;
+
+function MenuIcon(){
+    return(
+        <Button>
+            <Icon viewBox="0 0 24 24">
+                <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path>
```

Comment:
You could use [SVGr](https://react-svgr.com/) to convert SVGs to React components. No change is required as this as this is a preference.

Reply:
I'll take a look and implement it in future changes. :D